eng/pipelines/npm-tasks.yml

```diff
@@ -0,0 +1,31 @@
+trigger: none
```

Comment:
You can actually define these as pipeline parameters and they will show up as strongly typed options when someone manually queues a build.

Reply:
Sure. Thanks